src/SingleHacker/SingleHackerView.tsx

```diff
@@ -157,10 +161,20 @@ class SingleHackerView extends React.Component<
               />
             </Flex>
             <ViewPDFComponent hackerId={hacker.id} />
-            <hr />
-            <H2 color={theme.colors.grey}>Additional Information</H2>
-            <SHParagraph label="Why McHacks?" text={hacker.application.essay} />
-            <SHParagraph label="Comments" text={hacker.application.comments} />
+            {isAdmin && (
```

Comment:
We can also consider adding a `hidden` attribute to these components instead of hiding the component this way.

Reply:
yeah i like the idea. otherwise, we can use the `disabled` flag?

Reply:
What is the behavior of disabling a paragraph component?

Reply:
Oh, for some reason I thought this was the `Button` component. Disregard my last comment.

Reply:
I think in general, it might be better to not have the `hidden` attribute, cuz in this case, we want to hide the entire "Additional Information" section (including the header).

Reply:
We could try adding it in `SHField` later on to help with #369

Reply:
Let's do that, and then merge this PR

Reply:
Ah yeah I like the idea. I'll make some changes now.